src/api/seeders/20190703095400-reasons.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  up: queryInterface => queryInterface.bulkInsert('Reasons',
+    [
+      {
+        name: 'VIOLENT',
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        name: 'RAPE',
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        name: 'SEXUAL CONTENT',
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        name: 'HARRASSMENT',
+        createdAt: new Date(),
+        updatedAt: new Date()
+      },
+      {
+        name: 'I JUST DO NOT LIKE IT',
```

Comment:
lol @MCFrank16

Reply:
@MCFrank16 can please explain why is ``I JUST DO NOT LIKE``